android/src/main/java/com/bitmovin/player/reactnative/SourceModule.kt

```diff
@@ -76,12 +76,12 @@ class SourceModule(context: ReactApplicationContext) : BitmovinBaseModule(contex
         analyticsSourceMetadata: ReadableMap?,
         promise: Promise,
     ) = promise.unit.resolveOnUiThread {
+        if (sources.containsKey(nativeId)) {
+            return@resolveOnUiThread // key can be reused to access the same native instance (see NativeInstanceConfig)
```

Comment:
If the new TS instance has different configurations, they are ignored (which is fine IMO).
Should we check the configs here and show a warning if they do not match?

Reply:
Added a warning: https://github.com/bitmovin/bitmovin-player-react-native/pull/456/commits/6619a6ef28c496ebf4ed620019c7076c5e24a51d